examples/2d/elixir_euler_unstructured_quad_free_stream.jl

```diff
@@ -0,0 +1,61 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+
+initial_condition = initial_condition_constant
+boundary_conditions = BoundaryConditionDirichlet(initial_condition)
+
+###############################################################################
+# Get the DG approximation space
+
+polydeg = 6
+surface_flux = flux_hll
+solver = DGSEM(polydeg, surface_flux)
+
+###############################################################################
+# Get the curved quad mesh from a file
+
+mesh_file = joinpath(@__DIR__, "mesh_gingerbread_man.mesh")
```

Comment:
We might consider uploading this mesh to a gist or generating it on the fly using the new mesh generator in Julia. The same applies to all other mesh files of this PR.

Reply:
Yeah, I agree. We need a better way to manage this than the strategy of adding a new mesh if it is required for a new test.

Reply:
I uploaded a mesh to a gist and download it in an elixir in https://github.com/trixi-framework/Trixi.jl/blob/fc6beff8199e1970c75d79a4c9c3837e5ca7d644/benchmark/elixir_2d_euler_vortex_unstructured.jl#L14-L16